__tests__/static-pages.tsx

```diff
@@ -51,8 +51,190 @@ import {
 } from './__utils__'
 
+describe('serlo.org/terms', () => {
+  test('is in German at de.serlo.org/terms', async () => {
+    mockLegalPage('de/terms.md', 'Informationen für Weiternutzer')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'de',
+      pathname: '/terms',
+    })
+
+    expect(await response.text()).toEqual(
+      expect.stringContaining('Informationen für Weiternutzer')
+    )
+  })
+  test('is in English at en.serlo.org/terms', async () => {
+    mockLegalPage('en/terms.md', 'Terms of Use')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'en',
+      pathname: '/terms',
+    })
+
+    expect(await response.text()).toEqual(
+      expect.stringContaining('Terms of Use')
+    )
+  })
+})
+
+describe('serlo.org/imprint', () => {
+  test('is in German at de.serlo.org/imprint', async () => {
+    mockLegalPage('de/imprint.md', 'Impressum')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'de',
+      pathname: '/imprint',
+    })
+
+    expect(await response.text()).toEqual(expect.stringContaining('Impressum'))
+  })
+  test('is in English at en.serlo.org/imprint', async () => {
+    mockLegalPage('en/imprint.md', 'Imprint')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'en',
+      pathname: '/imprint',
+    })
+
+    expect(await response.text()).toEqual(expect.stringContaining('Imprint'))
+  })
+})
+
+describe('English version as default version', () => {
+  test('fr.serlo.org/terms gives the terms', async () => {
+    mockLegalPage('en/terms.md', 'Terms of Use')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'fr',
+      pathname: '/terms',
+    })
+
+    expect(await response.text()).toEqual(
+      expect.stringContaining('Terms of Use')
+    )
+  })
+  test('fr.serlo.org/imprint gives the imprint', async () => {
+    mockLegalPage('en/imprint.md', 'Imprint')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'fr',
+      pathname: '/imprint',
+    })
+
+    expect(await response.text()).toEqual(expect.stringContaining('Imprint'))
+  })
+  test('fr.serlo.org/privacy gives the privacy policy', async () => {
+    mockLegalPage('en/privacy.md', 'Privacy Policy')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'fr',
+      pathname: '/privacy',
+    })
+
+    expect(await response.text()).toEqual(
+      expect.stringContaining('Privacy Policy')
+    )
+  })
+})
+
+describe('serlo.org/privacy', () => {
+  test('is in German at de.serlo.org/privacy', async () => {
+    mockLegalPage('de/privacy.md', 'Datenschutzerklärung')
+
+    const response = await fetchTestEnvironment({
+      subdomain: 'de',
+      pathname: '/privacy',
+    })
+
+    expect(await response.text()).toEqual(
+      expect.stringContaining('Datenschutzerklärung')
+    )
+  })
+  test('link to the archive', async () => {
+    mockLegalPage('de/privacy.md', '<a href="/privacy/archive">Archiv</a>')
```

Comment:
```suggestion
    mockLegalPage('de/privacy.md', '')
```

Reply:
this was not necessary for the tests to work. Should I make this change anyways?